example-baseline/k8s/echo-env.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: echo-env
+  name: echo-env
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: example-baseline
+  strategy: {}
+  template:
+    metadata:
+      labels:
+        app: example-baseline
```

Comment:
This is wrong, because it doesn't match with the definition of the service selector.
I would rename it to:

`app: echo-env`